desktop/src/gui/theme.rs

```diff
@@ -171,10 +160,13 @@ impl ThemeController {
 }
 
 #[cfg(target_os = "linux")]
-fn scheme_to_theme(color_scheme: ColorScheme) -> Theme {
-    use crate::dbus::ColorScheme;
+fn scheme_to_theme(color_scheme: ashpd::desktop::settings::ColorScheme) -> Theme {
+    use ashpd::desktop::settings::ColorScheme;
     match color_scheme {
-        ColorScheme::Default => Theme::Light,
+        // We do not select the theme of window decorations,
```

Comment:
Could you please clarify what you mean by this?

Reply:
I mean... we don't select it "here", we don't _want_ to select it, we "can't" select it...?

Reply:
Note, on a Wayland compositor with no support for server-side decorations, I believe we have the `sctk` decorations, which does have a dark mode:
https://github.com/PolyMeilex/sctk-adwaita

Reply:
> Could you please clarify what you mean by this?
>
> I mean... we don't select it "here", we don't _want_ to select it, we "can't" select it...?

It means that we just don't do it. We *can* do it for sure, but I haven't checked whether it is feasible or how easy it is.

When we don't select it, it means we have to assume it's light in order not to desync decorations with GUI. There are apps that use dark mode without dark preference, and will use only light mode when strictly required (e.g. GNOME terminal, it's dark even when dark mode is disabled).

Note that this does not change the behavior, just adds a comment explaining the situation.

Reply:
Turns out it's fairly simple: https://github.com/ruffle-rs/ruffle/pull/18213